draft-pda-protocol.md

```diff
@@ -495,7 +495,7 @@ keys for a reasonable period after key changes in order to avoid
 rejecting reports.
 [[OPEN ISSUE: https://github.com/abetterinternet/prio-documents/issues/106]]
 
-### Upload Request
+### Report
```

Comment:
This hed seems a bit short. What's your thinking here.

Reply:
Only that it was more clear, but it's a separate change, so I'll just revert.